--- a/scripts/services/kernel
+++ b/scripts/services/kernel
@@ -135,6 +135,7 @@
       $SkipError = 1 if $ThisLine =~ /ERST: Error Record Serialization Table \(ERST\) support is initialized/;
       $SkipError = 1 if $ThisLine =~ /GHES: Generic hardware error source: \d+ notified via .* is not supported/;
       $SkipError = 1 if $ThisLine =~ /PCIe errors handled by (?:BIOS|OS)/;
+      $SkipError = 1 if $ThisLine =~ /RAS: Correctable Errors collector initialized\.$/;
       # These happen when kerberos tickets expire, which can be normal
       $SkipError = 1 if $ThisLine =~ /Error: state manager encountered RPCSEC_GSS session expired against NFSv4 server/ && $Ignore_rpcsec_expired;
       # filter out mount options

